build.gradle

```diff
@@ -2,6 +2,8 @@ group = projectGroup
 version = projectVersion
 description = projectDescription
 
+def myJarDestinationFolder = rootDir.path + '/production'
```

Comment:
usa il gradle.properties per settare questi valori, almeno ritroviamo tutte le configurazioni lì dentro

Reply:
Ho spostato il path tra gli altri path delle properties, ma la concatenazione alla rootDir rimane in quella posizione per evitare di duplicarla